source/main.css

```diff
@@ -1,9 +1,141 @@
-/* This is just a test CSS file. */
-html,
-body {
-    height: 100%;
+
+.info {
+    margin: 0 auto;
+    width: 80%;
+}
+
+div p {
+    text-align: left;
+    color: #000000;
+    font-size: 14px;
+    font-family: helvetica;
+}
+
+.photo-1 {
+    display: block;
+    max-width: 100%;
+    height: 900px;
+    /*margin: 0 auto;*/
+    background-image: url("./images/bb-photo1.png");
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: cover;
+    text-align: center;
+}
+
+.bb-photo {
+    display: block;
+    max-width: 100%;
+    margin: 0 auto;
+}
+
+.in-text {
+    margin: 0;
+    padding-left: 0.5%;
+    color: #ff5d00;
+    font-size: 15px;
+}
+
+.second-nav {
+    height: auto;
+    width: 100%;
+    text-align: center;
+    background-color: #000000;
+}
+
+.social-home {
+    display: inline-block;
+    width: auto;
+    margin: 0 auto;
+}
+
+.social-link {
+    margin: auto 60px;
+    text-align: center;
+    font-size: 22px;
+}
+
+/*.foot-step {
+    border-bottom: 2px solid #000000;
+    margin: 5px auto;
+    width: 520px;
+}*/
```

Comment:
I think you can apply these styles to a `hr` … maybe you don't need the `border-bottom` though?